qna/src/app/__init__.py

```diff
@@ -1,22 +1,21 @@
-from app.interface import BasicCLI, WebInterface
-from app.domain import UniversalEncoder, SentBERT, Doc2Vec
-from app.parser import parseQuestionsAnswersFromFile
-from app.parser import JsonLoader
+from .interface import BasicCLI, WebInterface
+from .domain import UniversalEncoder, SentBERT, Doc2Vec, T5
+from .parser import parseQuestionsAnswersFromFile
+from .parser import JsonLoader
+import os
 
 
 class App():
 
     def __init__(self, target_model, target_interface) -> None:
 
-        # Uncomment this if you want to create a new json file
-        '''
-        questions, answers = parseQuestionsAnswersFromFile(
-            'app/testfiles/help2002-2017.txt')
-        '''
-        json = JsonLoader('app/storage/questions2017_UE.json')
-        questions = json.read_data()
+        if len(os.listdir("app/storage")) == 0:
+            questions = parseQuestionsAnswersFromFile(
+                'app/testfiles/help2002-2017.txt')
```

Comment:
Looking at the definition of `parseQuestionsAnswersFromFile` it does not return anything, meaning that `questions` will always be `None`.

I believe this means that the app will break if `BERT` or `doc2vec` is selected.

Reply:
Yep fixing it now. I believe i removed it because of issues with basic CLI a while back. So i just wanted the json to be correct at that time.